task09/task09.2/task09.2/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task09._2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите N");
+            int n = int.Parse(Console.ReadLine());
+            int k = 0;
+            do
+            {
+            k++;
+            }
+            while (k*k <= n);
+            Console.WriteLine(k);
+            Console.ReadKey();
+        }
```

Comment:
Старайтесь структурировать код. И квадрат должен быть (строго) меньше _n_.
```suggestion
        static void Main()
        {
            Console.WriteLine("Введите N");
            int n = int.Parse(Console.ReadLine());
            int k = 0;
            do
            {
                k++;
            }
            while (k*k < n);
            
            Console.WriteLine(k);
            
            Console.ReadKey();
        }
```